Nature Publishing Group.js

```diff
@@ -354,7 +361,7 @@ function scrapeRIS(doc, url, next) {
 	if (!risURL) risURL = doc.evaluate('//li[@class="download-citation"]/a', doc, null, XPathResult.ANY_TYPE, null).iterateNext();
 	if (!risURL) risURL = doc.evaluate('//a[normalize-space(text())="Export citation" and not(@href="#")]', doc, null, XPathResult.ANY_TYPE, null).iterateNext();
 	if (!risURL) risURL = ZU.xpath(doc, '//ul[@data-component="article-info-list"]//a[@data-track-source="citation-download"]')[0];
-
+	if (!risURL) risURL = doc.querySelectorAll('a[data-track-action="download article citation"]')[0];
```

Comment:
For the first element you can also use `.querySelector` cf. https://developer.mozilla.org/de/docs/Web/API/Document/querySelector